test/StarcounterLinqUnitTests/Helpers/DataHelper.cs

```diff
@@ -0,0 +1,47 @@
+﻿using Starcounter;
+
+namespace StarcounterLinqUnitTests.Helpers
+{
+    public static class DataHelper
+    {
+        public static void ResetData()
+        {
+            Db.Transact(() =>
+            {
+                Db.SlowSQL("DELETE FROM StarcounterLinqUnitTests.Person");
```

Comment:
This query won't be executed very often, and it's speed is not important, that is why we can use `nameof` and `typeof` operators to compose the SQL query instead of string literals.

Reply:
Fixed.